README.md

```diff
@@ -28,8 +28,41 @@ This repo is Pantheon specific at the moment
 * Add Redis to the project
 ![Add redis to the project](https://user-images.githubusercontent.com/1062456/130299370-1e5564db-73dc-4ade-b086-5b7af27d7608.png)
 
+* Go to the Pantheon dashboard for your project.
+
+* Click on the Team tab.
+
+* Click on Add organization
+
+* Search for Kanopi Studios (Important: enter the full term (Kanopi Studios) to find this - if you just enter Kanopi, you will end up with the wrong group and things will not work).
+
+* Select and add Kanopi Studios.
+
+* Also add any Kanopi developers who will be working on the project as team members.
+
+#### Create project's Github repo
+* Go to [https://github.com/kanopi/drupal-starter](https://github.com/kanopi/drupal-starter)
+
+* Click on "Use this template" button.
+
+* Make the owner Kanopi and the repo private, then click "Create repository from template"
+
+* In the new repo, click on Settings and then the Manage Access tab.
+
+* Click on "Invite teams or people" button.
+
+* Search for and add Kanopi Studios and grant them Read access.
+
+* Click on "Add kanopicode to this repository"
+
```

Comment:
Can you please a comment about making the default branch protected too?   that would a good note

Reply:
@paulsheldrake I've added the steps about protecting the default branch. I think this is ready to merge, but let me know if you see anything missing or wrong.